src/api/procedures/modifySignerPermissions.ts

```diff
@@ -71,11 +76,9 @@ export async function prepareModifySignerPermissions(
  * @hidden
  */
 export function getAuthorization(
```

Comment:
Since you're not using the parameters at all, you can pass the result directly to the procedure instead of using this function